spec/support/jazz.rb

```diff
@@ -307,8 +307,23 @@ class InspectableKey < BaseObject
     field :is_flat, Boolean, null: false, method: :flat
   end
 
+  class HideEnsemble < GraphQL::Schema::TypeMembership
```

Comment:
`HideEnsemble` is a weird name for this class since it controls both hiding and showing? And because the `hide_ensemble` symbol used below doesn't actually map to this class name, which is confusing.

Reply:
Oops, thanks for pointing that out. I renamed it to `PerformingActVisibility`